apps/i18n/common/en_us.json

```diff
@@ -1423,7 +1423,6 @@
   "lessonExtrasDetails": "Your students won’t see this page unless you turn them on. You can turn Lesson Extras on by editing section details from your Teacher Dashboard.",
   "lessonName": "Lesson Name",
   "lessonNumber": "Lesson Number",
-  "lessonNumbered": "Lesson {lessonNumber}: {lessonName}",
```

Comment:
This string seems to be referenced from a few other places. Please don't delete it.